frontend/.storybook/main.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions" 
+  ],
+  "features": {
+    "interactionsDebugger": true,
+  },
+  "framework": "@storybook/react",
+  "core": {
+    "builder": "@storybook/builder-webpack5"
+  },
+  "typescript": {
+    "check": false,
+    "checkOptions": {},
+    "reactDocgen": false
+  },
+}
```

Comment:
Missing newline at EOF: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log
```suggestion
}

```